<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1606</width>
    <height>879</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>4</number>
    </property>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="CreateTemplateTab">
       <attribute name="title">
        <string>Create Template</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>-1</number>
        </property>
        <item row="0" column="0">
         <widget class="CreateTemplateWidget" name="create_template_widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="UseTemplateTab">
       <attribute name="title">
        <string>Use Template</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>-1</number>
        </property>
        <item row="0" column="0">
         <widget class="UseTemplateWidget" name="use_template_widget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1606</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_recent_templates">
     <property name="title">
      <string>Recent Templates</string>
     </property>
    </widget>
    <widget class="QMenu" name="menu_recent_packages">
     <property name="title">
      <string>Recent Packages</string>
     </property>
    </widget>
    <addaction name="actionOpen_Template"/>
    <addaction name="actionOpenPackage"/>
    <addaction name="separator"/>
    <addaction name="menu_recent_templates"/>
    <addaction name="menu_recent_packages"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Template"/>
    <addaction name="actionSave_Template_As"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Package"/>
    <addaction name="actionSave_Package_As_2"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenPackage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open Package</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave_Package">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save Package</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave_Package_As">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save Package As</string>
   </property>
  </action>
  <action name="actionUse_Template">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Template</string>
   </property>
  </action>
  <action name="actionCreate_Template">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Create Template</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close </string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSave_Template">
   <property name="text">
    <string>Save Template</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_Template">
   <property name="text">
    <string>Open Template</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPackages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Packages</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_clear_recent_templates">
   <property name="text">
    <string>Clear Recent Templates</string>
   </property>
  </action>
  <action name="action_clear_recent_packages">
   <property name="text">
    <string>Clear Recent Packages</string>
   </property>
  </action>
  <action name="actionSave_Template_As">
   <property name="text">
    <string>Save Template As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave_Package_As_2">
   <property name="text">
    <string>Save Package As</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CreateTemplateWidget</class>
   <extends>QWidget</extends>
   <header location="global">metaforge/widgets/createtemplatewidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UseTemplateWidget</class>
   <extends>QWidget</extends>
   <header location="global">metaforge/widgets/usetemplatewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
